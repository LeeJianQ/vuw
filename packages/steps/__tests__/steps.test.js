import VwSteps from "../src/steps";
import { mount } from "@vue/test-utils";

describe("vw-steps", () => {
  test("vw-steps-demo", () => {
    const wrapper = mount(VwSteps);
    expect(wrapper.html()).toContain('class="vw-steps-line"');
  });

  test("vw-steps-snapshot", () => {
    const wrapper = mount(VwSteps);
    expect(wrapper.vm.$el).toMatchSnapshot();
  });
});
