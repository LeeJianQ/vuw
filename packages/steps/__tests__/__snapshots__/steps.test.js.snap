// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vw-steps vw-steps-snapshot 1`] = `
<div
  class="vw-steps"
>
  <div
    class="vw-steps-list"
  >
    1
  </div>
   
  <div
    class="vw-steps-line"
  />
</div>
`;
